siac/rentercmd.go

```diff
@@ -96,6 +96,13 @@ have a reasonable number (>30) of hosts in your hostdb.`,
 		Run:   wrap(renterfilesdownloadcmd),
 	}
 
+	renterDownloadSectionCmd = &cobra.Command{
+		Use:   "downloadsection [path] [offset] [length] [destination]",
+		Short: "Download a chunk",
+		Long:  "Download a specific chunk from a previously uploaded file.",
+		Run:   wrap(renterfilesdownloadsectioncmd),
```

Comment:
@DavidVorick @RNabel thoughts? We can always add this later if there's a clear need for it.